wiz-admission-controller/templates/deployment.yaml

```diff
@@ -76,6 +76,10 @@ spec:
           - name: WIZ_ADMISSION_CONTROLLER_ENV
             value: {{ .Values.wizApiToken.clientEndpoint }}
           {{- end }}
+          {{- if .Values.HTTPS_PROXY }}
+          - name: HTTPS_PROXY
+            value: {{ .Values.HTTPS_PROXY | trim | quote }}
+          {{- end }}
```

Comment:
I think you might want to override a bunch of other values as well, such as `HTTP_PROXY` and the lowercase variants. You can see what @eyal-moscovici did for the diskanalyzer components (the charts are under `diskanalyzer-provisioner-helm`).